javascript/app.js

```diff
@@ -2,15 +2,215 @@ const menuBars = document.getElementById('menu-bars');
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_2.jpg',
+    urlDesktop: 'images/work_2_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://hafedefheij.github.io',
+    source: 'https://hafedefheij.github.io/mobile-portfolio/',
+  },
+  {
+    id: 2,
+    title: 'Tonic',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_3.jpg',
+    urlDesktop: 'images/work_3_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://hafedefheij.github.io/',
+    source: 'https://hafedefheij.github.io/mobile-portfolio/',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_4.jpg',
+    urlDesktop: 'images/work_4_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://hafedefheij.github.io/',
+    source: 'https://hafedefheij.github.io/mobile-portfolio/',
+  },
+];
+
+const worksLoadHandler = () => {
+  const works = document.getElementById('works');
+  worksArray.forEach((work) => {
+    const workCard = document.createElement('li');
+    workCard.classList.add('work-card');
+    workCard.innerHTML = `
+      <article  class='work-card__article'>
+        <div class='work-card__header'>
+          <picture>
+            <source srcset=${work.urlDesktop} media='(min-width: 768px)' />
+            <img
+              src=${work.urlMobile}
+              alt=${work.alt}
+            />
+          </picture>
+        </div>
+        <div class='work-card__body'>
```

Comment:
- [ ] Great effort creating the work section cards dynamically, but as suggested by the previous reviewer you should display the image and the text in the second and fourth card in a reversed manner in order to fit what we have in the Figma template.

Figma Design 👇 

![card](https://user-images.githubusercontent.com/78634841/209247112-7fc6a384-5a0a-495c-9b89-2c7215c73741.png)

 - Kindly follow the Figma image above to style and position your cards respectively.